src/test/java/com/puppycrawl/tools/checkstyle/AuditEventDefaultFormatterTest.java

```diff
@@ -104,6 +104,23 @@ public void testFormatModuleNameDoesNotContainCheckSuffix() {
     }
 
     @Test
+    public void testFormatModuleWithModuleId() {
+        final AuditEvent mock = PowerMockito.mock(AuditEvent.class);
```

Comment:
I saw every UT in this class used mock. I assumed there could be some reason so I just followed them.